src/mon/Monitor.cc

```diff
@@ -1054,7 +1054,11 @@ void Monitor::sync_obtain_latest_monmap(bufferlist &bl)
 
   dout(1) << __func__ << " obtained monmap e" << latest_monmap.epoch << dendl;
 
-  latest_monmap.encode(bl, quorum_features);
+  if (quorum_features) {
+    latest_monmap.encode(bl, quorum_features);
+  } else {
+    latest_monmap.encode(bl, CEPH_FEATURES_ALL);
+  }
```

Comment:
As far as i can tell, teh latest_monmap that we stash is only used locally--the encoded bl is never shared.  Which means we should just use CEPH_FEATURES_ALL all of the time.  In general quorum_features will always be 0 during sync...